source/extensions/filters/network/thrift_proxy/filters/ratelimit/BUILD

```diff
@@ -32,6 +32,7 @@ envoy_cc_extension(
     name = "config",
     srcs = ["config.cc"],
     hdrs = ["config.h"],
+    category = "envoy.thrift_proxy.filters",
```

Comment:
the source code has this as `envoy.thrift_proxy.filters` but in `extensions_build_config.bzl`  its `envoy.filters.network.thrift_proxy`

Reply:
Right.